meta-mender-demo/recipes-mender/pkcs11-provider/pkcs11-provider.bb

```diff
@@ -0,0 +1,17 @@
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://COPYING;md5=b53b787444a60266932bd270d1cf2d45 \
+                    file://LICENSES/Apache-2.0.txt;md5=3b83ef96387f14655fc854ddc3c6bd57"
+
+SRC_URI = "git://github.com/oleorhagen/pkcs11-provider;protocol=https;branch=main"
+
+PV = "0.2+git${SRCPV}"
+SRCREV = "${AUTOREV}"
```

Comment:
`AUTOREV` should not be used in permanent recipes, because it causes all users to probe the repositories on every build, so it's a performance problem. We should set the SHA instead.

Reply:
:+1: